docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
```

Comment:
> You actually probably don't want the quickstart sidenav as otherwise your reference doc will end up with the quickstarts. :)

Right. That's what I meant, but forgot "don't". Sorry :)